File: 30 Days of Code/Day 2 Operators/Solution.py
#!/bin/python3

import sys

if __name__ == "__main__":
    meal_cost = float( input( ).strip( ) )
    tip_percent = int( input( ).strip( ) )
    tax_percent = int( input( ).strip( ) )

    tip = meal_cost * ( tip_percent / 100 )
    tax = meal_cost * ( tax_percent / 100 )
    totalCost = int( round( meal_cost + tip + tax ) )

    #print( f"The total meal cost is {totalCost} dollars." )
    print( "The total meal cost is {0} dollars.".format( totalCost ) )

File: 30 Days of Code/Day 2 Operators/Solution.cs
using System;

class Solution {
    public static void Main( String[ ] args ) {
        double meal_cost = Convert.ToDouble( Console.ReadLine( ) );
        int tip_percent = Convert.ToInt32( Console.ReadLine( ) );
        int tax_percent = Convert.ToInt32( Console.ReadLine( ) );

        double tip = meal_cost * ( (double)tip_percent / 100 );
        double tax = meal_cost * ( (double)tax_percent / 100 );
        double totalCost = meal_cost + tip + tax;

        Console.WriteLine( $"The total meal cost is {Math.Round( totalCost )} dollars." );
    }
}